tests/unit/processes/ingest/test_u_of_m_process.py

```diff
@@ -1,7 +1,7 @@
 import pytest
 
 from mappings.core import MappingError
-from processes.UofM import UofMProcess
+from processes import UofMProcess
```

Comment:
Shouldn't this line be `from processes.ingest.u_of_m import UofMProcess.`

Reply:
ah because the __init__.py file in the processes folder imports the classes from the ingest file, this still works.

Reply:
Ah ok. I was just wondering since the other tests for these processes have that code line structure.

Reply:
ya some of the errors classes are not imported in the processes module, so it needs to be the full path

Reply:
Oh appreciate the clarification